src/utils/validation.ts

```diff
@@ -2,10 +2,11 @@ import chains from '@/config/chains'
 import { parsePrefixedAddress, sameAddress, isChecksummedAddress } from './addresses'
 import { safeFormatUnits, safeParseUnits } from './formatters'
 
-export const validateAddress = (address: string) => {
-  const ADDRESS_RE = /^0x[0-9a-f]{40}$/i
+const ADDRESS_RE = /^0x[0-9a-f]{40}$/i
+export const isAddress = (address: string) => ADDRESS_RE.test(address)
 
-  if (!ADDRESS_RE.test(address)) {
```

Comment:
Please restore it how it was if it's not used.

Reply:
Done